pkg/controller/actuator_test.go

```diff
@@ -312,9 +232,12 @@ var _ = Describe("actuator test", func() {
 			Expect(a.Reconcile(ctx, logger, ext)).To(Succeed())
 
 			// assert
-			envoyFilter := &istionetworkingClientGo.EnvoyFilter{}
-			Expect(k8sClient.Get(ctx, types.NamespacedName{Name: "acl-vpn", Namespace: istioNamespace1}, envoyFilter)).To(Succeed())
-			Expect(envoyFilter.Spec.MarshalJSON()).To(ContainSubstring("1.2.3.4"))
+			mr := &v1alpha1.ManagedResource{}
+			Expect(k8sClient.Get(ctx, types.NamespacedName{Name: ResourceNameSeed, Namespace: shootNamespace1}, mr)).To(Succeed())
+			secret := &corev1.Secret{}
+			Expect(k8sClient.Get(ctx, types.NamespacedName{Name: mr.Spec.SecretRefs[0].Name, Namespace: shootNamespace1}, secret)).To(Succeed())
+			Expect(secret.Data["seed"]).To(ContainSubstring("1.2.3.4"))
```

Comment:
Can you enhance the tests to test the actual content of the managed resource (at least for the new filter object)?